Presentation.Web/Models/API/V1/ItContractDTO.cs

```diff
@@ -83,5 +83,8 @@ public string ObjectOwnerFullName
         public IEnumerable<NamedEntityDTO> DataProcessingRegistrations { get; set; }
 
         public Guid Uuid { get; set; }
+        public int? CriticalityId { get; set; }
+        public string CriticalityName { get; set; }
+        public virtual CriticalityType CriticalityType { get; set; }
```

Comment:
wow is that the domain object you have there in the DTO?
that's a no go.. see how the other option types are handled

hint: To get automapper to work with this you must name it:

`CriticalityTypeName` and `CriticalityTypeId`
Then it will find the property and extract the name and id from it.. 
again, just look at how the others are used